integration-tests/src/test/scala/eu/ostrzyciel/jelly/integration_tests/io/IoSerDesSpec.scala

```diff
@@ -153,7 +178,8 @@ class IoSerDesSpec extends AnyWordSpec, Matchers, ScalaFutures, JenaTest:
             data.size should be > 0
             // In case we are leaving the default settings, RDF4J has no way of knowing if the data is
             // triples or quads, so it's going to default to quads.
-            val mayBeQuads = ser.name == "RDF4J" && preset.isEmpty
+            // Titanium just always does quads.
+            val mayBeQuads = ser.name == "RDF4J" && preset.isEmpty || ser.name == "Titanium"
```

Comment:
this IS nitpicking, but why not maybeQuads instead of mayBeQuads? should be easier to type and means roughly the same thing

Reply:
No, it's not "maybe quads", it's "may be quads" :D but that's old code anyway. This test will need a refactor anyway when we introduce test kits.